libpages/server.go

```diff
@@ -29,31 +30,70 @@ import (
 
 // ServerConfig holds configuration parameters for Server.
 type ServerConfig struct {
-	DomainWhitelist  []string
+	// If DomainWhitelist is non-nil and non-empty, only domains in the
+	// whitelist are served and others are blocked.
```

Comment:
Why do we have a whitelist? I don't see it used anywhere.

Reply:
It was in the initial version where I was imagining someone who host themselves might only want it to serve their own sites. We don't use that in `kbpagesd` yet, but I suppose when we make the "host yourself" doc we'll want that.